nttt/utilities.py

```diff
@@ -1,14 +1,15 @@
 from shutil import copy2, copystat, Error, ignore_patterns
 import os
 import os.path
-import codecs
+import eol
 
 def get_file(file_name):
-    with codecs.open(file_name, encoding='utf-8') as f:
-        return f.read()
+    with open(file_name, encoding='utf-8') as f:
+        (_, suggested_eol) = eol.eol_info_from_path(file_name)
```

Comment:
We need to remember the current EOL style in the file in order to use the same one when writing the content after modification

Reply:
Can you elaborate on this a bit further? New line 8 is very different from old line 8 and I cannot understand how this new style should be applied in other situations.